bin/reth/src/node/mod.rs

```diff
@@ -141,6 +141,10 @@ pub struct Command {
     /// Automatically mine blocks for new transactions
     #[arg(long)]
     auto_mine: bool,
+
+    /// Configuring block gas limit
+    #[arg(long, default_value = "30000000")]
+    block_gas_limit: u64,
```

Comment:
It does not make sense to expose this as it is network dependent, not something you can just configure. Thus, it should *only* be in the chainspec, not on the CLI. Please remove.

Reply:
@onbjerg Yes it seemed strange to me to be able to configure this on the CLI that's why I had made the first version. So this is like going back to commit 6ffe1a6c18e7e6bc5297fe1cdb99950861e636ff where the block gas limit was just exposed as a constant inside `ChainSpec`, correct?

Reply:
Yes, although it should be possible to set it in the actual chainspec JSON of course. So if I made a new chainspec for my own network, I should be able to increase or decrease it :)

Reply:
@onbjerg Should be ok now. Places where the default value is used remains at the default constant `ETHEREUM_BLOCK_GAS_LIMIT` while the other places directly take into account the `block_gas_limit` item of the `ChainSpec` structure.